backtrace-library/src/main/java/backtraceio/library/services/ReportExceptionTransformer.java

```diff
@@ -0,0 +1,130 @@
+package backtraceio.library.services;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import backtraceio.library.models.json.BacktraceReport;
+
+public class ReportExceptionTransformer {
+
+    /**
+     * Error trace attribute key
+     */
+    public static final String ErrorTraceAttribute = "error.trace";
+
+    /**
+     * Current error id attribute key
+     */
+    public static final String ErrorIdAttribute = "error.id";
```

Comment:
can be private

Reply:
I want to use it in other places. It's not a bad to keep it public.